lib/rabbit_client/bunny_client.rb

```diff
@@ -10,6 +10,34 @@ def self.publish_messages(url, exchange_name, exchange_type, messages)
       end
     end
 
+    def self.publish_message(url, exchange_name, exchange_type, message, routing_key, batch, origin)
```

Comment:
Could we use keyword arguments in this method?

Reply:
Sounds like a good idea, didnt think of that.